code/game/objects/effects/landmarks.dm

```diff
@@ -588,6 +588,62 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/awaystart) //Without this away mission
 	ruin_template = null
 	. = ..()
 
+
+/// Mob spawners, spawns a mob and deletes the landmark
+
+/obj/effect/landmark/mob_spawner
```

Comment:
The `"questionmark"`s below will be invisible unless you add `icon = 'icons/effects/spawner_icons.dmi'` here.

Reply:
Applied